tests/eclient/qemu+audio.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
Please, make this file executable.

Reply:
Probably, the correct variant here is to add `chmod` before the line https://github.com/lf-edge/eden/blob/94cb82fc33cf5d30287f39e537f98e1949997481/tests/eclient/Makefile#L55